app/Enums/ConferenceFilterEnum.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Enums;
+
+enum ConferenceFilterEnum: string {
+    case ALL = 'all';
+    case FUTURE = 'future';
+    case FAVORITE = 'favorites';
+    case DISMISSED = 'dismissed';
+    case OPENCFC = 'open_cfp';
+    case UNCLOSEDCFP = 'unclosed_cfp';
```

Comment:
```suggestion
    case OPEN_CFP = 'open_cfp';
    case UNCLOSED_CFP = 'unclosed_cfp';
```

Reply:
hi, I already remove the unnecessary file  please check it

thanks

Reply:
You still need to make the tests and linters pass.

Reply:
I already revert the commit the only file that i changed is only the BiosController